docs/spelling_wordlist.txt

```diff
@@ -23,6 +23,7 @@ coroutines
 datadog
 datadoghq
 datastore
+dbapi
```

Comment:
this is going to be a pain to maintain this list, is it worth it?

Reply:
If we want to automate spell checking, we'll need some form of a dictionary for our case. We could disable fail on warning but that would allow the scenario we had before where there were spelling errors in the public docs we publish.

Reply:
are there any public technology word lists we can pull in as a starting place?

I feel like this is going to get cumbersome anytime anyone wants to introduce a change to the docs.